tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -729,9 +717,9 @@ route/test_static_route.py::test_static_route_ecmp_ipv6:
 
 route/test_route_perf.py:
   skip:
-    reason: "M0 topo does not support test_route_perf"
+    reason: "M0/MX topo does not support test_route_perf"
     conditions:
-      - "topo_name in ['m0']"
+      - "topo_name in ['m0', 'mx']"
```

Comment:
Although the perf is not a concern for m0/mx topo since all managed devices is countable, we might need another test to make sure m0 can at least work as its expect

Reply:
Added